guides/v3.2.0/tutorial/autocomplete-component.md

```diff
@@ -229,14 +229,14 @@ export default Controller.extend({
   actions: {
     filterByCity(param) {
       if (param !== '') {
-        return this.get('store').query('rental', { city: param });
-        return this.get('store')
+        return this.store.query('rental', { city: param });
+        return this.store
           .query('rental', { city: param }).then((results) => {
             return { query: param, results: results };
           });
       } else {
-        return this.get('store').findAll('rental');
-        return this.get('store')
+        return this.store.findAll('rental');
+        return this.store
```

Comment:
Same as above

Reply:
same, displayed as diff in the rendered guides.